qchem/tests/test_active_space.py

```diff
@@ -52,5 +64,12 @@ def test_inconsistent_active_spaces(mol_name, n_act_electrons, n_act_orbitals, m
 
     r"""Test that an error is raised if an inconsistent active space is generated"""
 
+    molecule = MolecularData(filename=os.path.join(ref_dir, mol_name))
+
+    # with pytest.raises(ValueError, match=message_match):
+    #     qchem.active_space(mol_name, ref_dir, n_act_electrons, n_act_orbitals)
```

Comment:
Is this necessary to keep?

Reply:
The same here, thanks!. Forgot to clean this.